pkg/operator/ceph/file/controller.go

```diff
@@ -254,12 +286,62 @@ func (r *ReconcileCephFilesystem) reconcile(request reconcile.Request) (reconcil
 	logger.Debug("reconciling ceph filesystem store deployments")
 	reconcileResponse, err = r.reconcileCreateFilesystem(cephFilesystem)
 	if err != nil {
-		updateStatus(r.client, request.NamespacedName, k8sutil.ReconcileFailedStatus)
+		updateStatus(r.client, request.NamespacedName, cephv1.ConditionFailure, nil)
 		return reconcileResponse, err
 	}
 
-	// Set Ready status, we are done reconciling
-	updateStatus(r.client, request.NamespacedName, k8sutil.ReadyStatus)
+	// Enable mirroring if needed
+	// TODO: change me to 16.2.5 once it's out, in the mean this allows us to run the CI and validate the code
+	if r.clusterInfo.CephVersion.IsAtLeastPacific() {
```

Comment:
If it's not pacific (or 16.2.5) and there are mirroring settings, shall we fail the reconcile?

Reply:
Sounds good, ignoring works too